src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Sorter/Attributes/Date/LocalizableScopableSorterIntegration.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\PQB\Sorter\Date;
+
+use Pim\Bundle\CatalogBundle\tests\integration\PQB\AbstractProductQueryBuilderTestCase;
+use Pim\Component\Catalog\AttributeTypes;
+use Pim\Component\Catalog\Query\Sorter\Directions;
+
+/**
+ * Date sorter integration tests for localizable and scopable attribute.
+ *
+ * @author    Damien Carcel (damien.carcel@akeneo.com)
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class LocalizableScopableSorterIntegration extends AbstractProductQueryBuilderTestCase
+{
+    public function testSorterAscending()
+    {
+        $result = $this->executeSorter([[
+            'a_localizable_scopable_date',
+            Directions::ASCENDING,
+            ['locale' => 'fr_FR', 'scope' => 'tablet'],
+        ]]);
+        $this->assertOrder($result, ['product_four', 'product_three', 'product_two', 'product_one', 'empty_product']);
```

Comment:
seen IRL, this result is weird. `product_four` should be at the end of the result (as it has neither fr_FR locale, nor tablet scope)